src/braket/pennylane_plugin/__init__.py

```diff
@@ -15,6 +15,10 @@
     BraketAwsQubitDevice,
     BraketLocalQubitDevice,
 )
+from braket.pennylane_plugin.ahs_device import (
```

Comment:
The `noqa` will be needed to pass the linter
```suggestion
from braket.pennylane_plugin.ahs_device import (  # noqa: F401
```